### PR TITLE
Add header background - r1

### LINKED ISSUE #4: Add header background
At the moment, the header background is a colour. Have an option to add an image that's easy to swap to something else and works on multiple screen sizes.

### PR DESCRIPTION
**Pull request by ditsuke**
Fixes #4

## Changes
Adds a background image to the header, which can be swapped out for another image in the future.

### r1 cadddc8593eace7211e22937717e182876e9d987
image

### r2 769851d5b499437ecdf96922ae1583b11b9bdd7e (current)
image

## Authors
@ditsuke @harshsharma6401 

## Please, go through these steps before you submit a PR.

- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

### CODE REVIEW

File: _sass/profile.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .profile {
-    background-color: #133667 !important;
+    background: linear-gradient(to bottom, rgba(0, 0, 0, 0.05), black), no-repeat center/100% url('/assets/img/mlh-fellow.jpg'), #946c50;
```
Comment (gmelodie) [inline _sass/profile.scss:2]:
```suggestion
    background: linear-gradient(to bottom, rgba(0, 0, 0, 0.05), black), no-repeat center/100% url('/assets/img/background.jpg'), #946c50;
```
Renaming this to `background.jpg` would make it easier to understand and change in the future

**Review by gmelodie — CHANGES_REQUESTED** [review]
Small comment. Good work!

File: _sass/profile.scss (hunk 1 above)
Comment (ditsuke) [reply _sass/profile.scss]: Done and done, thanks!

**Review by yyjlincoln — APPROVED** [review]
It looks great!

**Comment by yyjlincoln** [comment]
Merging into `main`:
- All checks have passed, no conflicts
- Issues raised by @gmelodie have been fixed (so counting his `requesting changes` as `approved`)
- At least two approvals.

Great work!

**Comment by yyjlincoln** [comment]
Feel free to open another PR if you updated anything! Also, feel free to delete this branch if it's no longer required. Great work!

**Pull request merged**